src/course/syllabus/precourse-1/schedule.md

```diff
@@ -8,12 +12,28 @@ Practice using your Terminal to navigate your computer for the other tasks this
 
 Work through the [Semantic HTML workshop](/workshops/semantic-html/) to practice using more descriptive elements rather than divs and spans.
 
+## A11y
+
+Work through the [Learn Accessibility workshop](/workshops/learn-a11y/) to practice building sites for everyone.
+
 ## Git/GitHub
 
-Practice working with git and GitHub by pushing your solution to the HTML workshop to the FAC21 repository.
+Clone [this repository](https://github.com/fac22/project-collection) to your own profile and commit something each day. This will be a directory of the sites you create during the pre-apprenticeship.
+
+Practice working with git and GitHub by pushing your solution to the HTML workshop to this repository.
 
 ## Markdown
 
-Read [GitHub's Markdown introduction](https://guides.github.com/features/mastering-markdown/). Practice your Markdown by writing a `README.md` for your workshop solution. Try to include some headings, lists, images, and maybe even a table.
+Read [GitHub's Markdown introduction](https://guides.github.com/features/mastering-markdown/). Practice your Markdown by writing a `README.md` for your learning repository. Try to include some headings, lists, images, and maybe even a table.
+
+This should explain what you learnt, anything interesting you wanted to share etc.
+
+## Execute Program
+
+Work on [Execute Program](https://www.executeprogram.com/) every day. Start with the Modern JavaScript and Arrays sections.
+
+## Projects
+
+Build a website (or two) over the next two weeks. Focus on using Semantic HTML and ensuring the site has 100% in a Lighthouse Accessibility audit.
```

Comment:
I think we should try to be specific and targeted with project ideas here. I will try to think of some small projects that fit the stuff they're learning this week.

Reply:
Amazing, thanks. I'll have a think too.